lib/Compressor.js

```diff
@@ -77,8 +96,7 @@ export class Compressor extends Transformer {
   async _createTransformMaps({jsonldDocument}) {
     // initialize state
     this.contextMap = new Map();
-    this.termToId = new Map(KEYWORDS);
-    this.nextTermId = FIRST_CUSTOM_TERM_ID;
+    this.nextTermId = 100;
```

Comment:
Can we revert this to use the constant: `FIRST_CUSTOM_TERM_ID` ... so the semantics are better here?

```suggestion
    this.nextTermId = FIRST_CUSTOM_TERM_ID;
```